src/pages/[platform]/build-a-backend/server-side-rendering/nuxt/index.mdx

```diff
@@ -122,17 +122,14 @@ Example implementation:
 import type { CookieRef } from 'nuxt/app';
 import type { Schema } from '~/amplify/data/resource';
 import type { ListPaginateWithPathInput } from 'aws-amplify/storage';
-import type {
-  LibraryOptions,
-  FetchAuthSessionOptions
-} from '@aws-amplify/core';
 import {
   createKeyValueStorageFromCookieStorageAdapter,
   createUserPoolsTokenProvider,
   createAWSCredentialsAndIdentityIdProvider,
   runWithAmplifyServerContext
 } from 'aws-amplify/adapter-core';
 import { parseAmplifyConfig } from 'aws-amplify/utils';
+import type { FetchAuthSessionOptions } from 'aws-amplify/auth';
```

Comment:
nit, can we move this type import above the value imports?

Reply:
@josefaidt sorted & grouped all the imports.